packages/@webex/plugin-meetings/src/locus-info/controlsUtils.ts

```diff
@@ -119,4 +119,22 @@ ControlsUtils.getId = (controls: any) => {
   return null;
 };
 
+/**
+ * Extract the id from the record controls object
```

Comment:
issue: Can you update the description here to describe what this function is supposed to do